--- fftw-3.3.8/m4/ax_cc_maxopt.m4.orig	2018-05-24 14:03:22.000000000 +0200
+++ fftw-3.3.8/m4/ax_cc_maxopt.m4	2021-08-30 21:58:51.863783698 +0200
@@ -26,7 +26,7 @@
 AC_REQUIRE([AC_CANONICAL_HOST])
 
 # Try to determine "good" native compiler flags if none specified via CFLAGS
-if test "$ac_test_CFLAGS" != "set"; then
+if test "x$ac_test_CFLAGS" = "x"; then
   CFLAGS=""
   case $ax_cv_c_compiler_vendor in
     dec) CFLAGS="-newc -w0 -O5 -ansi_alias -ansi_args -fp_reorder -tune host"
