--- fftw-3.3/threads/Makefile.am.orig	2011-07-27 02:28:29.000000000 +0200
+++ fftw-3.3/threads/Makefile.am	2011-07-28 20:21:46.135119454 +0200
@@ -26,9 +26,11 @@
 vrank-geq1-rdft2.c f77api.c f77funcs.h
 libfftw3@PREC_SUFFIX@_threads_la_CFLAGS = $(PTHREAD_CFLAGS)
 libfftw3@PREC_SUFFIX@_threads_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
+libfftw3@PREC_SUFFIX@_threads_la_LIBADD = $(top_builddir)/libfftw3@PREC_SUFFIX@.la
 
 libfftw3@PREC_SUFFIX@_omp_la_SOURCES = api.c conf.c openmp.c	\
 threads.h dft-vrank-geq1.c ct.c rdft-vrank-geq1.c hc2hc.c	\
 vrank-geq1-rdft2.c f77api.c f77funcs.h
 libfftw3@PREC_SUFFIX@_omp_la_CFLAGS = $(OPENMP_CFLAGS)
 libfftw3@PREC_SUFFIX@_omp_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
+libfftw3@PREC_SUFFIX@_omp_la_LIBADD = $(top_builddir)/libfftw3@PREC_SUFFIX@.la -lgomp
--- fftw-3.3/Makefile.am.orig	2011-07-27 02:28:29.000000000 +0200
+++ fftw-3.3/Makefile.am	2011-07-28 20:18:54.851780384 +0200
@@ -13,8 +13,8 @@
 
 ACLOCAL_AMFLAGS=-I m4
 
-SUBDIRS=support $(GENFFT) kernel simd-support dft rdft reodft api	\
-threads libbench2 . tests mpi doc tools m4
+SUBDIRS=support $(GENFFT) kernel simd-support dft rdft reodft api .	\
+threads libbench2 tests mpi doc tools m4
 EXTRA_DIST=COPYRIGHT bootstrap.sh CONVENTIONS fftw.pc.in
 
 SIMD_LIBS =						\
